api/pkg/imagebuilder/imagebuilder.go

```diff
@@ -66,9 +66,14 @@ type imageBuilder struct {
 }
 
 const (
-	containerName      = "pyfunc-image-builder"
-	kanikoSecretName   = "kaniko-secret"
-	tickDurationSecond = 5
+	containerName    = "pyfunc-image-builder"
+	kanikoSecretName = "kaniko-secret"
+	// jobDeletionTickDurationMilliseconds is the interval at which the API server checks if a job has completed
+	jobCompletedTickDurationSecond = 5
```

Comment:
Nit: `jobCompletedTickDurationSecond` -> `jobCompletionTickDurationSecond` for consistency?

Reply:
Oops nice catch!